lib/actions/parse.js

```diff
@@ -102,29 +92,30 @@ var SUSPECT_PARSES = {
   // module.exports.foo = __webpack_require__(1);
   // module.exports.bar = __webpack_require__(2);
   // ```
-  MULTIPLE_EXPORT: function (src) {
+  MULTIPLE_EXPORTS: function (src) {
     var ast = parse(src, { sourceType: "module" });
     var reExportedReferences = getReExportedReferences(ast);
+    var snipped = "// ...";
 
+    // Check if we have one or less re-export (in which case we don't care.)
     if (_.keys(reExportedReferences).length < 2) { // eslint-disable-line no-magic-numbers
       return false;
     }
 
+    // Create a truncated version of output text with _just_ the re-export
+    // matches
     var srcLines = src.split("\n");
 
     var snippets = _.flow(
       _.values,
       _.uniqBy(_.identity),
-      _.map(function (location) {
-        return srcLines.slice(location.start.line - 1, location.end.line); // eslint-disable-line no-magic-numbers,max-len
+      _.map(function (loc) { // eslint-disable-next-line no-magic-numbers
+        return srcLines.slice(loc.start.line - 1, loc.end.line);
       }),
-      _.map(dedent),
```

Comment:
Ah, that makes sense.  I originally included the `dedent` so that deeply-indented exports lines wouldn't turn out to look like this:
```
    * 1: ./lib/mod-a.js
        * Matches: 1
            * MULTIPLE_EXPORTS:
                      module.exports = {
                        first: __webpack_require__(/*! ./first */ 2),
                        second: __webpack_require__(/*! ./second */ 3)
                      };


* ./lib/mod-b.js
    * Num Matches:         1
    * Num Files Matched:   1

    * 4: ./lib/mod-b.js
        * Matches: 1
            * MULTIPLE_EXPORTS:
                    module.exports.first = __webpack_require__(/*! ./first */ 2);
              // ...
                    module.exports.second = __webpack_require__(/*! ./second */ 3);
```
If you don't mind the wonky indentation on the report, no worries!